coroutines-interop/src/commonMain/kotlin/com/badoo/reaktive/coroutinesinterop/SingleFromCoroutine.kt

```diff
@@ -0,0 +1,26 @@
+package com.badoo.reaktive.coroutinesinterop
+
+import com.badoo.reaktive.single.Single
+import com.badoo.reaktive.single.single
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+fun <T> singleFromCoroutine(block: suspend () -> T): Single<T> =
```

Comment:
Add `fun completableFromCoroutine(block: suspend () -> Unit): Completable`?
Dispatcher as parameter?

Reply:
And you can also add `maybeFromCoroutine(block: suspend () -> T?)`.